projects/storefrontstyles/scss/cxbase/blocks/buttons.scss

```diff
@@ -171,3 +171,17 @@ button {
     opacity: 1;
   }
 }
+
+//theme calydon cx-base adjustments
+$theme: '' !default;
+@if $theme == 'calydon' {
```

Comment:
I would create a variable, named after the theme: `$calydon` which would be a string - `calydon`. so it could look like this: `@if $theme == $calydon {}`

Reply:
Thanks for the suggestion, I thought about that in the beginning, but later aligned it as to how Tobias explained the theme works, once you enable `calydon` or the theme that will come after this one, the variable kicks-in everywhere with the string declared in `projects/storefrontapp/src/styles.scss` otherwise the default theme remains.
 
I guess at some point we'll have to create a map with all the themes we will have or set them to base, but we'll see, for now `calydon` is still on alpha.

![Screen Shot 2019-09-19 at 9 59 51 AM](https://user-images.githubusercontent.com/15113219/65252273-aa79ce80-dac6-11e9-8e59-c29107714c99.png)